tutorials/nlp/Relation_Extraction-BioMegatron.ipynb

```diff
@@ -0,0 +1,601 @@
+   "source": [
+    "config.model.task_name = TASK\n",
+    "config.model.output_dir = WORK_DIR\n",
+    "config.model.dataset.data_dir = DATA_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "nB96-3sTc3yk"
+   },
+   "source": [
+    "## Building the PyTorch Lightning Trainer\n",
+    "\n",
+    "NeMo models are primarily PyTorch Lightning modules - and therefore are entirely compatible with the PyTorch Lightning ecosystem.\n",
+    "\n",
+    "Let's first instantiate a Trainer object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "1tG4FzZ4Ui60"
+   },
+   "outputs": [],
+   "source": [
+    "print(\"Trainer config - \\n\")\n",
+    "print(OmegaConf.to_yaml(config.trainer))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "knF6QeQQdMrH"
+   },
+   "outputs": [],
+   "source": [
+    "# lets modify some trainer configs\n",
+    "# checks if we have GPU available and uses it\n",
+    "cuda = 1 if torch.cuda.is_available() else 0\n",
+    "config.trainer.gpus = cuda\n",
+    "\n",
+    "config.trainer.precision = 16 if torch.cuda.is_available() else 32\n",
+    "\n",
+    "# for mixed precision training, uncomment the line below (precision should be set to 16 and amp_level to O1):\n",
+    "# config.trainer.amp_level = O1\n",
+    "\n",
+    "# remove distributed training flags\n",
+    "config.trainer.distributed_backend = None\n",
+    "\n",
+    "# setup max number of steps to reduce training time for demonstration purposes of this tutorial\n",
+    "config.trainer.max_steps = 5\n",
+    "\n",
+    "trainer = pl.Trainer(**config.trainer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8IlEMdVxdr6p"
+   },
+   "source": [
+    "## Setting up a NeMo Experiment\n",
+    "\n",
+    "NeMo has an experiment manager that handles logging and checkpointing for us, so let's use it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "8uztqGAmdrYt"
+   },
+   "outputs": [],
+   "source": [
+    "config.exp_manager.exp_dir = WORK_DIR\n",
+    "exp_dir = exp_manager(trainer, config.get(\"exp_manager\", None))\n",
+    "\n",
+    "# the exp_dir provides a path to the current experiment for easy access\n",
+    "exp_dir = str(exp_dir)\n",
+    "exp_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8tjLhUvL_o7_"
+   },
+   "source": [
+    "Before initializing the model, we might want to modify some of the model configs. For example, we might want to modify the pretrained BERT model and use [Megatron-LM BERT](https://arxiv.org/abs/1909.08053) or [AlBERT model](https://arxiv.org/abs/1909.11942):"
+   ]
```

Comment:
Is it relevant to your task? The notebook says BioMegatron. Maybe describe 2 available bioMegatron models?